app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -46,6 +47,10 @@ def ems_inv_to_hashes
 
       private
 
+      def get_network_interfaces
+        @data[:network_interfaces] = gather_data_for_this_region(@nis)
```

Comment:
@bronaghs I didn't use process_collection because that also sets "@data_index", which I thought was only meant for objects that actually have an underlying database table backing them up. Was that incorrect?